chat/src/test/java/net/md_5/bungee/api/chat/TranslatableComponentTest.java

```diff
@@ -9,7 +9,7 @@
     @Test
     public void testMissingPlaceholdersAdded()
     {
-        TranslatableComponent testComponent = new TranslatableComponent( "Test string with %s placeholders: %s", "2", "aoeu" );
+        TranslatableComponent testComponent = new TranslatableComponent( "Test string with %s placeholders: %s", 2, "aoeu" );
```

Comment:
%d for int placeholder? and perhaps also leave the string placeholder for variety?

Reply:
[Mojang uses %s for all placeholders](https://github.com/SpigotMC/BungeeCord/blob/master/chat/src/main/resources/mojang-translations/en_US.properties#L2092) regardless of what the variable type is, and it seems TextComponent only accepts string inputs (or inputs cast as string), so that should work.